adafruit_rfm69.py

```diff
@@ -727,8 +727,6 @@ def send(self, data, timeout=2., keep_listening=False,
         else:
         # Enter idle mode to stop receiving other packets.
             self.idle()
-        if timed_out:
-            raise RuntimeError('Timeout during packet send')
```

Comment:
suggest changing this  to;
return not timed_out

Reply:
also would need to update the doctsring.

Reply:
Making the timeout longer does not help.

Changing it to return (not timed_out) is a good alternative and will allow the caller to decide whether it needs further action or not.

Reply:
would you like to make that change in this PR or would you like me to incorporate it into PR #24